transport/rif_comms/cluster.py

```diff
@@ -0,0 +1,27 @@
+from raiden.tests.integration.network.transport.rif_comms.node import Node, Config
+
+
+class Cluster:
+    def __init__(self, nodes_to_clients: dict):
+        self.nodes_to_clients = nodes_to_clients
+        self.nodes = list()
+        for cluster_key in nodes_to_clients:
```

Comment:
the concept of "cluster key" is not clear by reading the code

Reply:
changed to `node_id`